src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,39 @@
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[] | null;
+};
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const preparedPeople = people?.map(person => ({
+    ...person,
+    mother: people.find(p => p.name === person.motherName),
+    father: people.find(p => p.name === person.fatherName),
+  }));
+
+  return (
+    <div>
+      <table
+        data-cy="peopleTable"
+        className="table is-striped is-hoverable is-narrow is-fullwidth"
+      >
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Sex</th>
+            <th>Born</th>
+            <th>Died</th>
+            <th>Mother</th>
+            <th>Father</th>
```

Comment:
Create array for column names and use `map` here

Reply:
> Create array for column names and use `map` here

Not fixed